extension/src/popup/components/WarningMessages/index.tsx

```diff
@@ -237,6 +259,48 @@ export const ScamAssetWarning = ({
     }
   }, [warningRef]);
 
+  const handleSubmit = async () => {
```

Comment:
spent a bit of time trying to find a good way of abstracting this away since similar code is used in ManageAssetRows, I couldn't find a clean way of doing since both are tied to UIs differently and use a bunch of vars from the outer component scope